ui/StatusQ/src/StatusQ/Popups/StatusAction.qml

```diff
@@ -14,6 +14,7 @@ Action {
     }
 
     property int type: StatusAction.Type.Normal
+    property bool visibleOnDisabled: false
```

Comment:
There's an easier way not requiring this extra property; `StatusMenu` has:
```qml
property bool hideDisabledItems: true
```

Reply:
Yeah that's exactly what the property does; `hideDisabledItems: false` will make sure that anything that has `enabled: false` will not get hidden but disabled - exactly what you need right? :)

Reply:
Yeah, that is different, fair enough